build-vm

```diff
@@ -662,66 +662,20 @@ vm_set_defaults() {
 }
 
 #
-# create file system and swap space, mount file system to $BUILD_ROOT
+# create swap space
 #
-vm_setup() {
-    vm_set_filesystem_type
-    vm_set_mount_options
-    echo "VM_ROOT: $VM_ROOT, VM_SWAP: $VM_SWAP"
```

Comment:
I think we should leave the echo call as it is instead of splitting it into two calls